helper/schema/shims_test.go

```diff
@@ -2024,11 +2024,11 @@ func TestShimSchemaMap_Diff(t *testing.T) {
 						RequiresNew: true,
 					},
 
-					"address": &terraform.ResourceAttrDiff{
-						Old:         "foo",
-						New:         "",
-						NewComputed: true,
-					},
+					// "address": &terraform.ResourceAttrDiff{
```

Comment:
Could you add a comment here that notes why this is commented out and under what circumstances we might uncomment it? (and the same for the other one below)

Reply:
thanks, I decided to get rid of rest of those once I confirmed that the new SimpleDiff did what I want, and just forgot these